light-zk.js/src/idls/verifier_program_one.ts

```diff
@@ -59,9 +59,14 @@ export type VerifierProgramOne = {
         "At successful verification protocol logic is executed."
       ],
       "accounts": [
+        {
+          "name": "verifierState",
+          "isMut": false,
+          "isSigner": false
+        },
         {
           "name": "signingAddress",
-          "isMut": true,
+          "isMut": false,
```

Comment:
whoops, something fishy going on here, need to investigate :eyes: 

I hope it's not a bug in light-anchor

Reply:
I have seen this before as well. It occured a couple times in a row. Then I wasn't able to reproduce it anymore.

Reply:
looks like isMut is inverted

Reply:
It was a silly mistake in the macro and how I was preserving fields, should be fixed now.